assets/built-in/transformers/dockerfile/dockerimagebuildscript/templates/buildandpushimages_multiarch.bat

```diff
@@ -29,6 +31,18 @@ if not %basename% == "scripts" (
 REM go to the parent directory so that all the relative paths will be correct
 cd {{ .RelParentOfSourceDir }}
 
+
+IF "%1"=="docker" (
+    shift
+    GOTO DOCKER_CONTAINER_RUNTIME
+)
+IF "%1"=="podman" (
+    shift
+    GOTO PODMAN_CONTAINER_RUNTIME
+)
+SET CONTAINER_RUNTIME={{ .ContainerRuntime }}
+GOTO DOCKER_CONTAINER_RUNTIME
```

Comment:
Because of this `GOTO` also, the code till line 64 will get skipped and even if the user specifies registry url/ registry namespace/ platforms while running the script, it will get ignored.